docs/dev/development/frontend.rst

```diff
@@ -8,14 +8,35 @@ serving.
 All of the static files are located in ``warehouse/static/`` and external
 libraries are found in ``package.json``.
 
+The static files are compiled and included in the
+``warehouse:docker-compose-static`` Docker image.
+
 
 Building
 --------
 
-Static files should be automatically built when ``make serve`` is running;
-however, you can trigger a manual build of them by installing
-`NodeJS 16.x <https://nodejs.org/en/download/releases/>`_, installing
-the dependencies using ``npm install`` and then running ``npm run build``.
+Static files should be automatically built when ``make serve`` is
+running; however, you can also manually run commands in the ``static``
+container:
+
+.. code-block:: console
+
+    $ # install dependencies
+    $ docker compose run --rm static npm install
+
+    $ # start a build
+    $ docker compose run --rm static npm run build
+
+
+Building outside of Docker
+--------------------------
+
+    Note: building outside of Docker is **not recommended** as it may
+    install platform-specific dependencies.
+
+Install `NodeJS 16.x <https://nodejs.org/en/download/releases/>`_,
```

Comment:
```suggestion
Install `NodeJS 20.x <https://nodejs.org/en/download/releases/>`_,
```

This has fallen out of sync with the actual version we build against

Reply:
Update, thank you.